cmd/pbm-agent/main.go

```diff
@@ -21,6 +23,7 @@ func main() {
 		pbmAgentCmd = pbmCmd.Command("run", "Run agent").Default().Hidden()
 
 		mURI = pbmAgentCmd.Flag("mongodb-uri", "MongoDB connection string").Envar("PBM_MONGODB_URI").Required().String()
+		dumpConns = pbmAgentCmd.Flag("parallel-collections", "Number of collections to dump in parallel").Envar("PBM_DUMP_PARALLEL_COLLECTIONS").Default(strconv.Itoa(runtime.NumCPU()/2)).Int()
```

Comment:
I suggest renaming `parallel-collections` flag to `dump-parallel-collections` since there is the same option for the mongorestore which maybe we would expose too